src/ascii_string.rs

```diff
@@ -14,6 +16,7 @@ use ascii_str::{AsciiStr, AsAsciiStr, AsAsciiStrError};
 
 /// A growable string stored as an ASCII encoded buffer.
 #[derive(Clone, Default, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
Shouldn't this also be annotated with `#[cfg_attr(feature = "serde", serde(transparent))]?